BuildingSync.xsd

```diff
@@ -1177,16 +1185,30 @@
           </xs:restriction>
         </xs:simpleType>
       </xs:element>
-      <xs:element name="Subsections" minOccurs="0">
+      <xs:element name="Sections" minOccurs="0">
         <xs:complexType>
           <xs:sequence>
-            <xs:element name="Subsection" maxOccurs="unbounded">
+            <xs:element name="Section" maxOccurs="unbounded">
               <xs:annotation>
-                <xs:documentation>Physical subsection of building for which geometric features are defined. May be one or many.</xs:documentation>
+                <xs:documentation>Physical section of building for which features are defined. May be one or many.</xs:documentation>
               </xs:annotation>
               <xs:complexType>
                 <xs:sequence>
                   <xs:element ref="auc:PremisesName" minOccurs="0"/>
+                  <xs:element name="SectionType" minOccurs="0">
+                    <xs:annotation>
+                      <xs:documentation>The type of section such as Whole Building data, Space Function data, or other types.</xs:documentation>
+                    </xs:annotation>
+                    <xs:simpleType>
+                      <xs:restriction base="xs:string">
+                        <xs:enumeration value="Whole Building"/>
```

Comment:
It might be a good idea to add annotations here, if that's possible. Is that possible?

Reply:
I can make a comment  
`<xs:enumeration value="Whole Building"/><!--this is a comment-->`

Or I can make the SectionType annotation define all of the enumerations in detail.

Reply:
I'm not sure that comments will come through in authoring tools like an annotation would. I'd vote for the SectionType annotation unless that makes it too awkward to deal with.